Mathlib/Tactic/Common.lean

```diff
@@ -60,7 +60,7 @@ import Mathlib.Tactic.InferParam
 import Mathlib.Tactic.Inhabit
 import Mathlib.Tactic.IrreducibleDef
 import Mathlib.Tactic.Lift
-import Mathlib.Tactic.Lint
+import Mathlib.Linter.Lint
```

Comment:
I think we should put this in a section under ProofWidgets just to keep this file organized (since it's no longer in alphabetical order), so that it looks something roughly like
```
...
-- Hopefully `lake` will be able to handle tests later.
import ProofWidgets

-- Import Mathlib linters.
import Mathlib.Linter.Lint

-- Now import all tactics defined in Mathlib that do not require theory files.
import Mathlib.Mathport.Rename
...
```
likely also with a change to the module doc
```
...
We include some commented out imports here, with an explanation of their theory requirements,
to save some time for anyone wondering why they are not here.

We also import theory-free linters, commands, and utilities which are useful to have low in the hierarchy.
-/
...
```
(Sorry, the [fact](https://github.com/orgs/community/discussions/4452) that github doesn't let me make suggestions on non-modified code is relevant once more... 😅)

Reply:
Thanks!  If I implemented your suggestions, assuming that I understood them correctly!